src/ballet/pack/fd_pack.c

```diff
@@ -1087,10 +1087,10 @@ fd_pack_schedule_impl( fd_pack_t          * pack,
       _mm512_stream_si512( (void*)(out->payload+1216UL), _mm512_load_epi64( cur->txn->payload+1216UL ) );
       /* Copied out to 1280 bytes, which copies some other fields we needed to
          copy anyway. */
-      FD_STATIC_ASSERT( offsetof(fd_txn_p_t, payload_sz   )+sizeof(((fd_txn_p_t*)NULL)->payload_sz   )<=1280UL, nt_memcpy );
-      FD_STATIC_ASSERT( offsetof(fd_txn_p_t, requested_cus)+sizeof(((fd_txn_p_t*)NULL)->requested_cus)<=1280UL, nt_memcpy );
-      FD_STATIC_ASSERT( offsetof(fd_txn_p_t, executed_cus )+sizeof(((fd_txn_p_t*)NULL)->executed_cus )<=1280UL, nt_memcpy );
-      FD_STATIC_ASSERT( offsetof(fd_txn_p_t, flags        )+sizeof(((fd_txn_p_t*)NULL)->flags        )<=1280UL, nt_memcpy );
+      FD_STATIC_ASSERT( offsetof(fd_txn_p_t, payload_sz                     )+sizeof(((fd_txn_p_t*)NULL)->payload_sz                     )<=1280UL, nt_memcpy );
+      FD_STATIC_ASSERT( offsetof(fd_txn_p_t, pack_cu.requested_execution_cus)+sizeof(((fd_txn_p_t*)NULL)->pack_cu.requested_execution_cus)<=1280UL, nt_memcpy );
+      FD_STATIC_ASSERT( offsetof(fd_txn_p_t, pack_cu.non_execution_cus      )+sizeof(((fd_txn_p_t*)NULL)->pack_cu.non_execution_cus      )<=1280UL, nt_memcpy );
```

Comment:
This can just be one of the struct elements of the union to save a line, since that includes both uints